org.o20.word.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "org.o20.word",
```

Comment:
so this certainly isn't going to be the "right" app-id unless you own the o20.org website. see https://github.com/flathub/flathub/wiki/App-Requirements#Application-ID for more.

Reply:
Would io.gitlab.o20.word be better?